examples/with-elasticsearch/pages/index.js

```diff
@@ -0,0 +1,234 @@
+import Head from 'next/head'
+import { connectToElasticsearch } from '../lib/elasticsearch'
+
+export default function Home({ isConnected }) {
+  return (
+    <div className="container">
+      <Head>
+        <title>Create Next App</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        <h1 className="title">
+          Welcome to <a href="https://nextjs.org">Next.js with Elasticsearch!</a>
+        </h1>
+        {isConnected ? (
+          <h5 className="subtitle">You are connected to Elasticsearch</h5>
+        ) : (
+          <h5 className="subtitle">
+            You are NOT connected to Elasticsearch. Check the <code>README.md</code>{' '}
+            for instructions.
+          </h5>
+        )}
+
+        <p className="description">
+          Get started by editing <code>pages/index.js</code>
+        </p>
+
+        <div className="grid">
+          <a href="https://nextjs.org/docs" className="card">
+            <h3>Documentation &rarr;</h3>
+            <p>Find in-depth information about Next.js features and API.</p>
+          </a>
+
+          <a href="https://nextjs.org/learn" className="card">
+            <h3>Learn &rarr;</h3>
+            <p>Learn about Next.js in an interactive course with quizzes!</p>
+          </a>
+
+          <a
+            href="https://github.com/vercel/next.js/tree/master/examples"
+            className="card"
+          >
+            <h3>Examples &rarr;</h3>
+            <p>Discover and deploy boilerplate example Next.js projects.</p>
+          </a>
+
+          <a
+            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
+            className="card"
+          >
+            <h3>Deploy &rarr;</h3>
+            <p>
+              Instantly deploy your Next.js site to a public URL with Vercel.
+            </p>
+          </a>
+        </div>
+      </main>
+
+      <footer>
+        <a
+          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Powered by{' '}
+          <img src="/vercel.svg" alt="Vercel Logo" className="logo" />
+        </a>
+      </footer>
+
+      <style jsx>{`
```

Comment:
Could this use CSS Modules instead like create-next-app?

Reply:
@leerob moved the CSS into modules just like create-next-app.